src/python/shallow_water_visualization.py

```diff
@@ -33,6 +33,62 @@ def inverse(self, value):
 
         return np.interp(value, x, y, left=-np.inf, right=np.inf)
 
+<<<<<<< HEAD
```

Comment:
Not sure if you're still working on resolving conflicts, but your last commit may have included incomplete merge.

Reply:
Yep, pushed the commit a little early. Still working on cleaning this up.